### DISCUSSION TITLE
Multidiffusion feature

### DISCUSSION THREAD

**Discussion by katarzynasornat** [huggingface/diffusers#7927] [Q&A]
Hi all! Does diffusers library support currently regional prompt for multidiffusion or only panorama? Could somebody share an example how to run region based multidiffusion with masks?

Thank you

**Comment by tolgacangoz** [comment]
Hi Kasia! There is a community pipeline called [RegionalPromptingStableDiffusionPipeline](https://github.com/huggingface/diffusers/blob/main/examples/community/README.md#regional-prompting-pipeline); could you examine it?

**Reply by katarzynasornat** [reply]
Hi @standardAI thanks for your answer! I have tried to run the example: https://github.com/huggingface/diffusers/tree/main/examples/community#sample-code

but unfortunately I am getting some error while loading the pipeline. Please find the [gist](https://colab.research.google.com/gist/katarzynasornat/3e71f4d7168ffc727b95c587fb9658b0/regional_prompting_example.ipynb]) and the error below

AttributeError                            Traceback (most recent call last)
[<ipython-input-16-ea6deb5db909>](https://localhost:8080/#) in <cell line: 1>()
----> 1 pipe = RegionalPromptingStableDiffusionPipeline.from_pretrained(MODEL_PATH, vae = vae)

5 frames
[/content/diffusers/src/diffusers/pipelines/pipeline_loading_utils.py](https://localhost:8080/#) in _fetch_class_library_tuple(module)
    731     # register the config from the original module, not the dynamo compiled one
    732     not_compiled_module = _unwrap_model(module)
--> 733     library = not_compiled_module.__module__.split(".")[0]
    734 
    735     # check if the module is a pipeline module

AttributeError: 'bool' object has no attribute '__module__'

Any help is very welcome! :)

**Reply by tolgacangoz** [reply]
Hmm, it seems that this pipeline needs refactoring: https://github.com/huggingface/diffusers/issues/6984
You can try it yourself, or just downgrade `diffusers` to a lower version for now.

**Reply by katarzynasornat** [reply]
I will try to refactor it (I have seen this discussion that somebody have already started but without a success) but before doing that (as I am quite new to diffusers and not a pro in Deep Learning but I am very fast student:)) I wanted to downgrade to a lower version. Unfortunately I had another error which made me impossible to run and see if/how the pipeline works.

Below I enclosed my [gist](https://colab.research.google.com/gist/katarzynasornat/f0574c6f27e171fe81870b91cd97eea2/regional_prompt_pipeline_downgrade.ipynb). Would you be kind to take a look what could go wrong? Otherwise hard to improve the pipeline which may not working anyway.
